src/RJP.MultiUrlPicker/Models/MultiUrls.cs

```diff
@@ -39,12 +40,15 @@ public MultiUrls(string propertyData)
 
         private void Initialize(JArray data)
         {
+            // Create UmbracoHelper here, because we (probably) need it more than once
+            var umbracoHelper = data.Count > 1 ? new UmbracoHelper(UmbracoContext.Current) : null;
```

Comment:
Since we only need the `UmbracoHelper` if the link uses content or media, creating it here and not using it could actually decrease performance. We take that chance if we have more than 1 link...